tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_course_metadata.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for SAPSF course metadata exporters.
+"""
+
+from __future__ import absolute_import, unicode_literals, with_statement
+
+import unittest
+
+import ddt
+import mock
+import responses
+from integrated_channels.sap_success_factors.exporters.course_metadata import SapSuccessFactorsCourseExporter
+from pytest import mark, raises
+
+from test_utils import factories
+from test_utils.fake_enterprise_api import EnterpriseMockMixin
+
+
+@mark.django_db
+@ddt.ddt
+class TestSapSuccessFactorsCourseExporter(unittest.TestCase, EnterpriseMockMixin):
+    """
+    Tests for the ``SapSuccessFactorsCourseExporter`` class.
+    """
+
+    def setUp(self):
+        self.enterprise_customer = factories.EnterpriseCustomerFactory()
+        self.mock_ent_courses_api_with_pagination(
+            enterprise_uuid=str(self.enterprise_customer.uuid),
+            course_run_ids=['course-v1:edX+DemoX+Demo_Course_1']
+        )
```

Comment:
@UmanShahzad Same note here about moving this down to keep mock definitions together.

Reply:
:+1: